tailwind.config.ts

```diff
@@ -3,60 +3,80 @@ import type { Config } from "tailwindcss"
 const config = {
   darkMode: ["class"],
   content: [
-    './pages/**/*.{ts,tsx}',
-    './components/**/*.{ts,tsx}',
-    './app/**/*.{ts,tsx}',
-    './src/**/*.{ts,tsx}',
-	],
+    "./src/**/*.{ts,tsx}",
+    // TODO: remove after migration
+    "./tailwind/**/*.tsx",
+  ],
   prefix: "",
   theme: {
-    container: {
-      center: true,
-      padding: "2rem",
-      screens: {
-        "2xl": "1400px",
-      },
-    },
     extend: {
+      fontFamily: {
+        heading: "var(--font-inter)",
+        body: "var(--font-inter)",
+        monospace: "var(--font-mono)",
+      },
+      lineHeight: {
+        "6xs": "1.1",
+        "5xs": "1.15",
+        "4xs": "1.2",
+        "3xs": "1.25",
+        "2xs": "1.3",
+        xs: "1.4",
+        sm: "1.5",
+        base: "1.6",
+      },
       colors: {
-        border: "hsl(var(--border))",
-        input: "hsl(var(--input))",
-        ring: "hsl(var(--ring))",
-        background: "hsl(var(--background))",
-        foreground: "hsl(var(--foreground))",
         primary: {
-          DEFAULT: "hsl(var(--primary))",
-          foreground: "hsl(var(--primary-foreground))",
-        },
-        secondary: {
-          DEFAULT: "hsl(var(--secondary))",
-          foreground: "hsl(var(--secondary-foreground))",
+          DEFAULT: "var(--primary)",
+          highContrast: "var(--primary-highContrast)",
+          lowContrast: "var(--primary-lowContrast)",
```

Comment:
Not sure about this. Do you know how these are compiled? are they going to be kebab case? or are they going to keep the same?
This: `bg-primary-lowContrast` vs `bg-primary-low-contrast`. I would prefer to keep them as kebab now that we are using this convention. What do you think?

Reply:
Compiling retains pascalCase. Good with me flipping them to kebab. That feels more natural here.

Reply:
I've updated the naming here